### PR TITLE
22.3 fb clean up onprc ehr.xml

### PR DESCRIPTION
**Pull request by jonesgaohsu**
#### Rationale
Clean up of onprc_ehr.xml data per instructions from labkey to allow deletion of folders and workbooks
#### Related Pull Requests
Closer review of the recommendations from Josh revealed I had not compelted all the needed actions.  Needed to remove the onprc_ehr.XX.sql that created the files.  
#### Changes
*

### CODE REVIEW

File: onprc_ehr/resources/queries/ehr_lookups/roomsAssignment.sql [hunk 1]
```diff
@@ -34,7 +35,7 @@ select
   cast(a.project.name || ' (' || a.project.investigatorId.lastname || '): ' || cast(count(distinct a.id.curLocation.cage) as varchar) as varchar) as projectCageTotal,
   count(distinct a.id) as totalAnimals
 
-from study.assignment a
+from finance_Study.assignment a
```
Comment (labkey-jeckels) [inline onprc_ehr/resources/queries/ehr_lookups/roomsAssignment.sql]: This change seemed unrelated and accidentally included. Please remove.

File: onprc_ehr/resources/schemas/onprc_ehr.xml [hunk 2]
```diff
@@ -763,72 +652,8 @@
             <column columnName="container"/>
         </columns>
     </table>
-    <table tableName="eIACUC_PRIME_VIEW_ANIMAL_GROUPS" tableDbType="TABLE">
```
Comment (labkey-jeckels) [inline onprc_ehr/resources/schemas/onprc_ehr.xml]: This table, and the other eIACUC* tables, should be retained. The problematic ones are the ones that start with PRIME_VIEW

File: onprc_ehr/resources/schemas/onprc_ehr.xml [hunk 3]
```diff
@@ -763,72 +652,8 @@
...
-            <column columnName="Food_Withheld"/>
-            <column columnName="Water_Withheld"/>
-            <column columnName="Food_Water_Withheld_Description"/>
-            <column columnName="Food_Water_Withheld_Justification"/>
-            <column columnName="Food_Water_Withheld_Adverse_Consequences"/>
-            <column columnName="Death_As_Endpoint_Number_of_Animals"/>
-            <column columnName="Death_As_Endpoint_Justification"/>
-        </columns>
-    </table>
-    <table tableName="eIACUC_PRIME_VIEW_SURGICAL_PROCS" tableDbType="TABLE">
```
Comment (labkey-jeckels) [inline onprc_ehr/resources/schemas/onprc_ehr.xml]: Should not be removed.

**Review by labkey-jeckels — CHANGES_REQUESTED** [review]
We can't completely delete onprc_ehr-18.10-20.101.sql. We should keep the file but delete the statements that are creating the problematic project, protocol, and PRIME_VIEW tables.

File: onprc_ehr/resources/schemas/onprc_ehr.xml (hunk 2 above)
Comment (jonesgaohsu) [reply onprc_ehr/resources/schemas/onprc_ehr.xml]: So should I change in intellij and then resubmit?

Comment (labkey-jeckels) [reply onprc_ehr/resources/schemas/onprc_ehr.xml]: @jonesgaohsu yes please. From a first glance, your other followup commits here look correct.

Comment (jonesgaohsu) [reply onprc_ehr/resources/schemas/onprc_ehr.xml]: Thanks for the support on this

Comment (jonesgaohsu) [reply onprc_ehr/resources/schemas/onprc_ehr.xml]: I pulled down a fresh copy and made the needed corrections and will verify in test instance once the build is complete.  I believe the needed modifications and removals were made

File: onprc_ehr/resources/schemas/dbscripts/sqlserver/onprc_ehr-18.10-20.101.sql [hunk 4]
```diff
@@ -114,115 +114,6 @@ GO
...
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- * 2/17/2018 Jones ga
- * This script creates the ONPRC_EHR.PRIME_VIEW_PROTOCOL_Processing Dataset which is populated by the ETL Process
- *
- */
 
 CREATE TABLE [onprc_ehr].[PRIME_VIEW_PROTOCOL_Processing](
```
Comment (labkey-jeckels) [inline onprc_ehr/resources/schemas/dbscripts/sqlserver/onprc_ehr-18.10-20.101.sql]: This one should get deleted too

**Review by labkey-jeckels — CHANGES_REQUESTED** [review]

File: onprc_ehr/resources/schemas/dbscripts/sqlserver/onprc_ehr-18.10-20.101.sql (hunk 4 above)
Comment (jonesgaohsu) [reply onprc_ehr/resources/schemas/dbscripts/sqlserver/onprc_ehr-18.10-20.101.sql]: OK will remove and reload

Comment (jonesgaohsu) [reply onprc_ehr/resources/schemas/dbscripts/sqlserver/onprc_ehr-18.10-20.101.sql]: Copncentrating on clean up of onprc_ehr-18..10-20.101 and removing all reference to eiACUC processsing, this was 3 year old code will be updating the build in the next 30 minutes

Comment (jonesgaohsu) [reply onprc_ehr/resources/schemas/dbscripts/sqlserver/onprc_ehr-18.10-20.101.sql]: Push completed My review shows all references to eIACUC processing has been removed

File: onprc_ehr/resources/web/onprc_ehr/panel/AssignmentFilterType.js [hunk 5]
```diff
@@ -65,7 +65,7 @@ Ext4.define('ONPRC_EHR.panel.AssignmentFilterType', {
                 store: {
                     type: 'labkey-store',
                     schemaName: 'onprc_ehr',
-                    sql: 'SELECT distinct lastname FROM onprc_ehr.investigators WHERE lastname is not null',
+                    sql: 'SELECT distinct lastname FROM onprc_ehr.investigators WHERE enddate is null',
```
Comment (labkey-jeckels) [inline onprc_ehr/resources/web/onprc_ehr/panel/AssignmentFilterType.js]: This change seems unrelated. Is it something you want to include here?

Comment (jonesgaohsu) [reply onprc_ehr/resources/web/onprc_ehr/panel/AssignmentFilterType.js]: NO let me remove it from the build

Comment (jonesgaohsu) [reply onprc_ehr/resources/web/onprc_ehr/panel/AssignmentFilterType.js]: File restored to original state and submitted

Comment (labkey-jeckels) [reply onprc_ehr/resources/web/onprc_ehr/panel/AssignmentFilterType.js]: @jonesgaohsu you added a comment but didn't actually back out the change to the SQL.

**Review by labkey-jeckels — CHANGES_REQUESTED** [review]

**Comment by jonesgaohsu** [comment]
Made the change on line 68 to return the code to original code

**Review by labkey-jeckels — APPROVED** [review]
